src/thresholds.service.js

```diff
@@ -55,19 +60,28 @@ class ThresholdsService {
       weeksOfStock = plan.weeksOfStock
     }
 
-    let thresholds = Object.keys(weeklyLevels).reduce((index, product) => {
-      index[product] = Object.keys(weeksOfStock).reduce((productThresholds, threshold) => {
-        productThresholds[threshold] = Math.round(weeklyLevels[product] * weeksOfStock[threshold])
+    let thresholds = Object.keys(weeklyLevels).reduce((index, productId) => {
+      index[productId] = Object.keys(weeksOfStock).reduce((productThresholds, threshold) => {
+        const level = weeklyLevels[productId] * weeksOfStock[threshold]
+        const product = find(products, isId.bind(null, productId))
+        if (!product || !product.presentation) {
+          return productThresholds
```

Comment:
I wondered about this as well. I originally was throwing an error, but used this to fit with existing style.

I think throwing would be the least surprising thing to do, but is probably better suited for another PR?